packages/integration-tests/test/simpleStartStandardExitTest.js

```diff
@@ -58,116 +58,143 @@ describe('simpleStartStandardExitTest.js', function () {
     }
   })
 
-  describe('childchain transaction exit', function () {
-    const INTIIAL_ALICE_AMOUNT = web3.utils.toWei('.001', 'ether')
-    const INTIIAL_BOB_RC_AMOUNT = web3.utils.toWei('.1', 'ether')
-    const TRANSFER_AMOUNT = web3.utils.toWei('0.0002', 'ether')
-    let aliceAccount
-    let bobAccount
-
-    beforeEach(async function () {
-      aliceAccount = rcHelper.createAccount(web3)
-      bobAccount = rcHelper.createAccount(web3)
-      await Promise.all([
-        // Give some ETH to Alice on the child chain
-        faucet.fundChildchain(
+  describe('hitting watcher-info service only', function () {
+    before(async function () {
+      childChain = new ChildChain({
+        watcherUrl: config.watcher_url,
+        watcherProxyUrl: config.watcher_proxy_url,
+        plasmaContractAddress: config.plasmaframework_contract_address
+      })
+    })
+
+    testChildchainTransactionStartExit()
+  })
+
+  describe('hitting both watcher-security and watcher-info service', function () {
+    before(async function () {
+      childChain = new ChildChain({
+        watcherUrl: config.watcher_url,
+        watcherSecurityUrl: config.watcher_security_url,
+        watcherProxyUrl: config.watcher_proxy_url,
+        plasmaContractAddress: config.plasmaframework_contract_address
+      })
+    })
+
+    testChildchainTransactionStartExit()
```

Comment:
I think it is better to make `testChildchainTransactionStartExit` accept a childchain instance
eg.
`function testChildchainTransactionStartExit(cc) {
}
`
and that function use the `cc` so you don't need to override the childchain variable